plansys2_problem_expert/include/plansys2_problem_expert/ProblemExpertClient.hpp

```diff
@@ -59,36 +59,19 @@ class ProblemExpertClient : public ProblemExpertInterface
   bool removeInstance(const plansys2::Instance & instance);
   std::optional<plansys2::Instance> getInstance(const std::string & name);
 
-  std::vector<plansys2_msgs::msg::Node> getPredicateNodes();
```

Comment:
I think you also want to remove the functions declared on line 52-55 of this file.

Reply:
See commit 25dc11d.